salt/client/mixins.py

```diff
@@ -433,7 +433,7 @@ def print_async_event(self, suffix, event):
         # more general, since this will get *really* messy as
         # people use more events that don't quite fit into this mold
         if suffix == 'ret':  # for "ret" just print out return
-            salt.output.display_output(event['return'], None, self.opts)
+            salt.output.display_output(event['return'], self.opts.get('out', None), self.opts)
```

Comment:
uhm if i remember well we (with @jacksont)j disabled this as for runners, we expect raw results directly from the runner itself to be ouputted to stdout, if i understand well what you are trying to do there, im not ok :D